megalinter/Linter.py

```diff
@@ -1076,6 +1090,43 @@ def get_total_number_errors(self, stdout: str):
         else:
             return 1
 
+    # Find number of warnings in linter stdout log
+    def get_total_number_warnings(self, stdout: str):
```

Comment:
Please could you factorize and create a method get_total_number_issues(self, level, stdout) ? 
Depending on level you would use cli_lint_errors_count or cli_lint_warnings_count, etc... :)

Reply:
I've been thinking about the architecture of a change like this. Because errors are found when a return_code > 0 is used, it only runs sometimes. However, because warnings can still show when the return code is 0, if we want warnings to surface we either have to seperate the warning code (what I have right now) or look for both warnings and errors even if the return_code is 0. What do you think?